x-pack/plugins/maps/kibana.json

```diff
@@ -8,10 +8,12 @@
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "maps"],
   "requiredPlugins": [
+    "lens",
```

Comment:
I don't think we should make lens depend on maps since lens does not use any thing provided from the maps plugin. But I will move map's lens dependencies to optionalPlugins since maps app works just fine without lens plugin.

Reply:
Actually, I can not move lens to required plugins since maps code imports some constants now. Either way, lens should not depend on maps